### ISSUE TITLE
cross_validation returns a warning when using tibbles

### ISSUE THREAD

**Issue by cphaarmeyer** [facebook/prophet#1128]
When `df` is a tibble, calling `cross_validation` on the model `m <- prophet(df)` returns the warning:

> Unknown or uninitialised column: 'y'.

I tracked this down to the function `setup_dataframe` and the call `any(is.infinite(df$y))` where `df$y` returns this warning since `y` never exists.

**Comment by bletham** [comment]
Thanks for reporting this. This issue can be produced with this code:
```
library(prophet)
library(tibble)
library(readr)

df <- read_csv('example_wp_log_peyton_manning.csv')
m <- prophet(df)

df.cv <- cross_validation(m, initial = 730, period = 365, horizon = 365, units = 'days')
```

I'm not really sure what's happening here since the issue does not occur if df is a regular dataframe and not a tibble. I suspect there is a command used in cross validation that is overriden for tibbles and behaves slightly differently?
It does seem at least that despite the warning it is producing results that match those produced with a regular dataframe, but this definitely needs to be tracked down to figure out why the warning is coming up at all.

**Comment by cphaarmeyer** [comment]
I think it comes down to this difference (quote from `vignette("tibble")`):

> Tibbles are also stricter with $. Tibbles never do partial matching, and will throw a warning and return NULL if the column does not exist

There is apparently no influence on the functionality just an irritating warning.

**Comment by bletham** [comment]
This is fixed in https://github.com/facebook/prophet/commit/b372feb31fb8cb4dfe79b47225b0ab96589e4692

**Comment by bletham** [comment]
The fix is pushed to PyPI in v0.6
